models/repo.go

```diff
@@ -1963,8 +1963,17 @@ func GetUserRepositories(userID int64, private bool, page, pageSize int, orderBy
 	}
 	sess.Limit(pageSize, (page-1)*pageSize)
 
-	repos := make([]*Repository, 0, pageSize)
-	return repos, sess.Find(&repos)
+	repos = make([]*Repository, 0, pageSize)
+
+	if err = sess.Find(&repos); err != nil {
+		return
+	}
+
+	if err = repos.loadAttributes(x); err != nil {
```

Comment:
I found serval references of `GetUserRepositories` but maybe not every place need `loadAttributes`. So could you move the `LoadAttributes` to the user profile router function?

Reply:
@lunny this should not be done in backport but first fixed in master

Reply:
@lunny Done.

Reply:
@appleboy as @lafriks said, this PR should not be back ported to v1.4 since it's a performance optimization.